tests/foreman/cli/test_remoteexecution.py

```diff
@@ -344,7 +345,8 @@ def test_positive_run_job_effective_user(
                 'organization-id': module_org.id,
             }
         )
-        assert 'Permission denied' in out
+        if not rex_contenthost.os_version.major == 7 and is_open('SAT-30898'):
```

Comment:
Ruff complains about this one. Could you fix that?
```suggestion
        if rex_contenthost.os_version.major != 7 and is_open('SAT-30898'):
```